packages/live-share-media/src/VolumeLimiter.ts

```diff
@@ -3,59 +3,60 @@
  * Licensed under the Microsoft Live Share SDK License.
  */
 
- import { TimeInterval } from '@microsoft/live-share';
- import { IMediaPlayer } from './IMediaPlayer';
+import { TimeInterval } from '@microsoft/live-share';
+import { IMediaPlayer } from './IMediaPlayer';
 
 export enum LevelType { fixed, percentage }
-
 export class VolumeLimiter {
     private readonly _player: IMediaPlayer;
+    private readonly _rampDuration = new TimeInterval(500);
 
-    // Settings
-    private readonly _duration = new TimeInterval(2000);
-    private readonly _rampDownDuration = new TimeInterval(500);
-    private readonly _rampUpDuration = new TimeInterval(500);
-    private _level: number = 0.1
+    private _selectedVolume = 1.0;
+    private _limited = false;
+    private _level = 0.1
     private _levelType: LevelType = LevelType.fixed;
-
-    // Running state
-    private _running = false;
-    private _endTime = 0;
     private _startTime = 0;
     private _startVolume = 0;
-    private _inRamp = false;
+    private _running = false;
 
     constructor(player: IMediaPlayer) {
         this._player = player;
+        this._player.volume = this._selectedVolume;
     }
 
     /**
-     * Time, in seconds, that the volume should be lowered.
-     * 
+     * The selected volume.
+     *
      * @remarks
-     * Applications should periodically call `lowerVolume()` to keep the volume lowered. 
+     * Expressed as a value between 0.0 and 1.0. The default value is 1.0.
+     * Can be used for things like volume sliders.
      */
-    public get duration(): number {
-        return this._duration.seconds;
+    public get selectedVolume(): number {
+        return this._selectedVolume;
     }
 
-    public set duration(value: number) {
-        this._duration.seconds = Math.abs(value);
+    public set selectedVolume(value: number) {
```

Comment:
Name like `maxVolume` or `maximumVolume` might be better. Also needs a comment since it is a public property.